src/main/java/com/iexec/worker/docker/DockerService.java

```diff
@@ -58,62 +58,87 @@ public DockerService(WorkerConfigurationService workerConfigService,
      * @return a Docker client
      */
     public DockerClientInstance getClient() {
-        if (dockerClientInstance == null) {
-            dockerClientInstance = getAuthForRegistry(DEFAULT_REGISTRY_ADDRESS)
-                    .map(defaultAuth ->
-                            DockerClientFactory.getDockerClientInstance(defaultAuth.getUsername(),
-                                    defaultAuth.getPassword()))
-                    .orElse(DockerClientFactory.getDockerClientInstance());
+        if (dockerClientInstance != null) {
+            return dockerClientInstance;
         }
-        return dockerClientInstance;
+        Optional<RegistryCredentials> registryCredentials = dockerRegistryConfiguration
+                .getRegistryCredentials(DEFAULT_REGISTRY_ADDRESS);
```

Comment:
Do we need DEFAULT_REGISTRY_ADDRESS for this method? I think we already have getClient(String imageName) & getClient(String registryAddress, ..).
Before any pullImage, we must use getClient(String imageName), but we don't need elsewhere IMO

Reply:
Remove, thanks!